### PR TITLE
docs: card-update

### PR DESCRIPTION
**Pull request by Ildest**
New try

### CODE REVIEW

**Review by Ildest — COMMENTED** [review]
Reviewed and fixed missing line breaks

**Comment by Ildest** [comment]
Please review - have fixed missing line breaks and it looked Ok in loclahost: http://localhost:6006/?path=/docs/komponenter-card--docs

File: packages/react/src/components/Card/Card.mdx [hunk 1]
```diff
@@ -63,30 +86,43 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
...
 <Canvas of={CardStories.Composed} />
 
-## Retningslinjer
+## Retningslinjer for `Card`
+
+Vi bruker kort når vi vil trekke oppmerksomheten til brukeren mot en funksjon.
+De fungerer godt til interaksjon, men vi bruker dem bare hvis vi har tenkt å tilby ekstra funksjonalitet med gruppert innhold.
 
-### Egnet til
+Kort passer veldig godt til responsive løsninger. Vi kan ha flere kort på en
```
Comment (mrosvik) [inline packages/react/src/components/Card/Card.mdx:94]:
```suggestion

```

File: packages/react/src/components/Card/Card.mdx [hunk 2]
```diff
@@ -63,30 +86,43 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
...
+side, men pass på at de oppfattes som rolige elementer på siden, og ikke tar
+oppmerksomheten og avbryter hvordan brukerne leser informasjonen på siden.
```
Comment (mrosvik) [inline packages/react/src/components/Card/Card.mdx:96]:
```suggestion

```

File: packages/react/src/components/Card/Card.mdx [hunk 3]
```diff
@@ -63,30 +86,43 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
...
 
-- Presentere relatert informasjon i en oversiktlig format.
-- Vise produktinformasjon, brukeranmeldelser, eller blogginnlegg.
-- Gruppere funksjoner eller tjenester i en app eller på en nettside.
+Passer til å gruppere innhold eller funksjonalitet som du vil skille ut fra
+resten av innholdet.
 
-### Ikke egnet til
+Passer ikke til å
```
Comment (mrosvik) [inline packages/react/src/components/Card/Card.mdx:101]:
```suggestion

```

File: packages/react/src/components/Card/Card.mdx [hunk 4]
```diff
@@ -63,30 +86,43 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
...
 
-- Vise store mengder tekst eller data som kan være overveldende.
-- Erstatte navigasjonskomponenter.
-- Skjule viktig informasjon som brukeren må se umiddelbart.
+- presentere hovedinnholdet i tjenesten, men heller innhold som supplerer og skal være en del av helheten
+- være kun et visuelt blikkfang, det bør være interaksjon på kort
+- gi statusmeldinger
```
Comment (mrosvik) [inline packages/react/src/components/Card/Card.mdx:105]:
```suggestion

```

File: packages/react/src/components/Card/Card.mdx [hunk 5]
```diff
@@ -63,30 +86,43 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
...
+  {' '}
```
Comment (mrosvik) [inline packages/react/src/components/Card/Card.mdx:106]:
```suggestion

```

File: packages/react/src/components/Card/Card.mdx [hunk 6]
```diff
@@ -63,30 +86,43 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
...
+- å blande for mange forskjellige typer innhold i et enkelt kort
 
-<br />
-## Tekst i komponenten
+## Tekst i komponenten{' '}
 
-Sørg for at overskriftene og innholdet i kortet er klare og informative.
-Unngå vage titler og gi brukerne en god forståelse av hva de kan forvente å finne
-i hvert kort.
+Unngå lange tekster på kortene. Tenkt nøye gjennom hva du trenger å si og skriv en kort og presis overskrift, underoverskrift og eventuell
```
Comment (mrosvik) [inline packages/react/src/components/Card/Card.mdx:115]:
```suggestion
Unngå lange tekster på kortene. Tenk nøye gjennom hva du trenger å si og skriv en kort og presis overskrift, underoverskrift og eventuell
```

File: packages/react/src/components/Card/Card.mdx [hunk 7]
```diff
@@ -63,30 +86,43 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
...
+
+[Ikke funnet noe her]
+
+### Kilder
+
+designsystem.
+dk jokul.no
 
 <br />
+```
```
Comment (mrosvik) [inline packages/react/src/components/Card/Card.mdx:128]:
```suggestion

```

File: packages/react/src/components/Card/Card.mdx [hunk 8]
```diff
@@ -63,30 +86,43 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
...
-Sørg for at overskriftene og innholdet i kortet er klare og informative.
-Unngå vage titler og gi brukerne en god forståelse av hva de kan forvente å finne
-i hvert kort.
+Unngå lange tekster på kortene. Tenkt nøye gjennom hva du trenger å si og skriv en kort og presis overskrift, underoverskrift og eventuell
+brødtekst.
+
+## Tilgjengelighet
+
+[Ikke funnet noe her]
+
```
Comment (mrosvik) [inline packages/react/src/components/Card/Card.mdx:121]:
```suggestion

```

File: packages/react/src/components/Card/Card.mdx [hunk 9]
```diff
@@ -63,30 +86,43 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
 
 <Canvas of={CardStories.Composed} />
 
-## Retningslinjer
+## Retningslinjer for `Card`
+
+Vi bruker kort når vi vil trekke oppmerksomheten til brukeren mot en funksjon.
+De fungerer godt til interaksjon, men vi bruker dem bare hvis vi har tenkt å tilby ekstra funksjonalitet med gruppert innhold.
```
Comment (mrosvik) [inline packages/react/src/components/Card/Card.mdx:92]:
```suggestion
Dersom kortet er en link med bilde, så skal **ikke** bildet være informativt, og `alt` attributtet skal være tomt.
```

**Review by mimarz — APPROVED** [review]
Getting the changes merged so we can continue work on this in other PRs

File: packages/react/src/components/Card/Card.mdx [hunk 10]
```diff
@@ -1,40 +1,35 @@
...
+
+
+
 ## Eksempler
 
-### `Card.Media`
+`Card.Media`
 
-Bruk `Card.Media` dersom du ønsker å ha bilder eller video i kortene.
+Bruk`Card.Media` hvis du vi ha bilder eller video i kortene.
```
Comment (Ildest) [inline packages/react/src/components/Card/Card.mdx:32]: Rette vi til vil

File: packages/react/src/components/Card/Card.mdx [hunk 11]
```diff
@@ -63,30 +61,28 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
 
 <Canvas of={CardStories.Composed} />
 
-## Retningslinjer
+## Retningslinjer for `Card`
 
-### Egnet til
+Vi bruker kort når vi vil trekke oppmerksomheten til brukeren mot en funksjon.
+Dersom kortet er en link med bilde, så skal **ikke** bildet være informativt, og `alt` attributtet skal være tomt.
```
Comment (Ildest) [inline packages/react/src/components/Card/Card.mdx:67]:
```suggestion
Hvis kortet er en lenke med bilde, skal **ikke** bildet være informativt, og `alt`-attributtet skal være tomt.
```

File: packages/react/src/components/Card/Card.mdx [hunk 12]
```diff
@@ -63,30 +61,28 @@ Eksempler på hvordan andre komponenter kan brukes til å komponere sammen i et
...
+- å blande for mange forskjellige typer innhold i et enkelt kort
 
-<br />
-## Tekst i komponenten
+## Tekst i komponenten{' '}
 
-Sørg for at overskriftene og innholdet i kortet er klare og informative.
-Unngå vage titler og gi brukerne en god forståelse av hva de kan forvente å finne
-i hvert kort.
+Unngå lange tekster på kortene. Tenk nøye gjennom hva du trenger å si og skriv en kort og presis overskrift, underoverskrift og eventuell
```
Comment (Ildest) [inline packages/react/src/components/Card/Card.mdx:82]:
```suggestion
Unngå lange tekster på kortene. Tenk nøye gjennom hva du trenger å si, og skriv en kort og presis overskrift, underoverskrift og eventuell
```

File: packages/react/src/components/Card/Card.mdx [hunk 13]
```diff
@@ -44,16 +39,19 @@ Bruk `Card.Media` dersom du ønsker å ha bilder eller video i kortene.
...
+Link](https://nextjs.org/docs/pages/api-reference/components/link) og [React
+Router Link](https://reactrouter.com/en/main/components/link)
 
 <Canvas of={CardStories.LinkCard} />
 
 ### Farger
 
-Viser forskjellige varianter av Card komponenten, inkludert nøytral,
-subtil, og andre stiler.
+Viser forskjellige varianter av `Card`-komponenten, inkludert nøytral, subtil, og andre stiler.
```
Comment (Ildest) [inline packages/react/src/components/Card/Card.mdx:54]:
```suggestion
Viser forskjellige varianter av `Card`-komponenten, som nøytral, dempet, og andre stiler.
```

File: packages/react/src/components/Card/Card.mdx [hunk 14]
```diff
@@ -44,16 +39,19 @@ Bruk `Card.Media` dersom du ønsker å ha bilder eller video i kortene.
...
 ### Link Card
 
-Bruk `as` til å endre kortet til en lenke. Vær oppmerksom på gyldig html [innhold](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#technical_summary) og innholdet da skjermleser alt samlet.
-
-Hover og aktivt styling blir kun aktivert dersom `href` er definert. Dette er for å støtte "Link" komponenter fra tredjeparts bibliotek som [NextJS Link](https://nextjs.org/docs/pages/api-reference/components/link) og [React Router Link](https://reactrouter.com/en/main/components/link)
+Bruk `as` til å endre kortet til en lenke. Vær oppmerksom på gyldig html
+[innhold](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#technical_summary)
+og innholdet, fordi skjermlesere leser opp alt samlet. Hover og aktivt styling
+blir kun aktivert dersom du har definert `href`. Dette er for å støtte "Link"
+komponenter fra tredjeparts bibliotek som [NextJS
```
Comment (Ildest) [inline packages/react/src/components/Card/Card.mdx:46]:
```suggestion
komponenter fra tredjepartsbibliotek som [NextJS
```

File: packages/react/src/components/Card/Card.mdx [hunk 15]
```diff
@@ -44,16 +39,19 @@ Bruk `Card.Media` dersom du ønsker å ha bilder eller video i kortene.
 
 ### Link Card
 
-Bruk `as` til å endre kortet til en lenke. Vær oppmerksom på gyldig html [innhold](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#technical_summary) og innholdet da skjermleser alt samlet.
-
-Hover og aktivt styling blir kun aktivert dersom `href` er definert. Dette er for å støtte "Link" komponenter fra tredjeparts bibliotek som [NextJS Link](https://nextjs.org/docs/pages/api-reference/components/link) og [React Router Link](https://reactrouter.com/en/main/components/link)
+Bruk `as` til å endre kortet til en lenke. Vær oppmerksom på gyldig html
+[innhold](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#technical_summary)
+og innholdet, fordi skjermlesere leser opp alt samlet. Hover og aktivt styling
+blir kun aktivert dersom du har definert `href`. Dette er for å støtte "Link"
```
Comment (Ildest) [inline packages/react/src/components/Card/Card.mdx:45]:
```suggestion
blir kun aktivert dersom du har definert `href`. Dette er for å støtte "Link"-
```

File: packages/react/src/components/Card/Card.mdx (hunk 10 above)
Comment (Ildest) [inline packages/react/src/components/Card/Card.mdx:32]:
```suggestion
Bruk`Card.Media` hvis du vil ha bilder eller video i kortene.
```

File: packages/react/src/components/Card/Card.mdx [hunk 16]
```diff
@@ -1,40 +1,35 @@
...
 
 # Card
 
-Med `Card` kan du presentere informasjon på en strukturert og oversiktlig måte. Det er ideelt for å vise innhold som er relatert, men som kan deles inn i forskjellige seksjoner.
-
-**Vær oppmerksom på:**
-
-- Kort bør ikke inneholde for mye informasjon som kan overvelde brukeren.
-- Dersom kortet er en link med bilde, så skal **ikke** bildet være informativt, og `alt` attributtet skal være tomt.
+Med `Card` kan vi fremheve informasjon eller oppgaver som hører sammen. Kortene finnes i flere varianter og kan inneholde overskrift, underoverskrift, et område for annen tekst og knapper og eller lenker
```
Comment (Ildest) [inline packages/react/src/components/Card/Card.mdx:8]:
```suggestion
Med `Card` kan vi fremheve informasjon eller oppgaver som hører sammen. Kortene finnes i flere varianter og kan inneholde overskrift, underoverskrift, et område for annen tekst og knapper og eller lenker.
```

**Review by Ildest — COMMENTED** [review]
Gjennomgått og lagt inn forslag

**Pull request merged**